packages/react-vapor/src/components/tab/Tab.tsx

```diff
@@ -10,6 +10,7 @@ export interface ITabOwnProps {
     title: string;
     disabled?: boolean;
     tooltip?: string;
+    children?: React.ReactNode;

```

Comment:
I think you don't need to add it to the interface since React.Component already adds it to the props object